.gitops/charts/traffic-court-test-values.yaml

```diff
@@ -7,14 +7,14 @@ global:
 
 arc-dispute-api:
   image:
-    tag: "1.47.2"
+    tag: "1.48.1"
```

Comment:
Should we update this test-values.yaml as well at this point since we are only releasing to dev environment right now?

Reply:
Good call. Removed the changes to the test file.